docs/api/cases/cases-api-find-connectors.asciidoc

```diff
@@ -0,0 +1,51 @@
+[[cases-api-find-connectors]]
+== Find connectors API
+++++
+<titleabbrev>Find connectors</titleabbrev>
+++++
+
+Retrieves a paginated subset of <<action-types,connectors>>.
+
+In particular, only the connectors that are supported for use in cases are
+returned. Refer to the list of supported external incident management systems in
+<<add-case-connectors>>.
+
+=== Request
+
+`GET <kibana host>:<port>/api/cases/configure/connectors/_find`
```

Comment:
We should also document the path that takes a space ID, and document that if the space is not provided in the URL, the default space is used.

```
GET <kibana host>:<port>/s/<space_id>/api/cases/configure/connectors/_find
```

Reply:
Thanks! I've added that info in 1f36fd9da172bf319fa6acd1b31a11cf46624d9a